src/pages/home.js

```diff
@@ -1,6 +1,25 @@
+import { useSelector,useDispatch } from 'react-redux'
+import DropDown from '../Components/Buttons/DropDown';
+import OutlinedButton from '../Components/Buttons/OulinedButton';
+import PlainButton from '../Components/Buttons/PlainButton';
+import { ActionType } from '../Redux/Contants/action-type';
```

Comment:
Use imports in following order
1. Framework defaults
2. 3rd party imports
3. Your own imports

In above, it will be like

```
import { useSelector,useDispatch } from 'react-redux'
import { ActionType } from '../Redux/Contants/action-type';

 import DropDown from '../Components/Buttons/DropDown';
 import OutlinedButton from '../Components/Buttons/OulinedButton';
 import PlainButton from '../Components/Buttons/PlainButton';
```